tests/vmi_multus_test.go

```diff
@@ -834,37 +844,19 @@ var _ = Describe("SRIOV", func() {
 		// properly, or via in-PF switching for VFs (works for some NIC models)
 		pingThroughSriov := func(cidrA, cidrB string) {
 			// start peer machines with sriov interfaces from the same resource pool
-			vmi1 := getSriovVmi([]string{"sriov-link-enabled"})
-			vmi2 := getSriovVmi([]string{"sriov-link-enabled"})
-
-			// Explicitly choose different random mac addresses instead of relying on kubemacpool to do it:
-			// 1) we don't at the moment deploy kubemacpool in kind providers
-			// 2) even if we would do, it's probably a good idea to have the suite not depend on this fact
-			//
-			// This step is needed to guarantee that no VFs on the PF carry a duplicate MAC address that may affect
-			// ability of VMIs to send and receive ICMP packets on their ports.
-			mac1, err := tests.GenerateRandomMac()
-			Expect(err).ToNot(HaveOccurred())
-
-			mac2, err := tests.GenerateRandomMac()
-			Expect(err).ToNot(HaveOccurred())
-
-			vmi1.Spec.Domain.Devices.Interfaces[1].MacAddress = mac1.String()
-			vmi2.Spec.Domain.Devices.Interfaces[1].MacAddress = mac2.String()
-
-			startVmi(vmi1)
-			startVmi(vmi2)
-			waitVmi(vmi1)
-			waitVmi(vmi2)
+			vmi1 := createSriovVMI([]string{"sriov-link-enabled"})
+			vmi2 := createSriovVMI([]string{"sriov-link-enabled"})
 
 			// manually configure IP/link on sriov interfaces because there is
 			// no DHCP server to serve the address to the guest
 			configInterface(vmi1, "eth1", cidrA, "#")
 			configInterface(vmi2, "eth1", cidrB, "#")
 
 			// now check ICMP goes both ways
-			pingVirtualMachine(vmi1, cidrToIP(cidrA), "#")
-			pingVirtualMachine(vmi2, cidrToIP(cidrB), "#")
+			err = testPingFromVMI(vmi1, cidrToIP(cidrA), "#")
+			Expect(err).ToNot(HaveOccurred(), "VMI %s should be reachable", vmi1.Name)
```

Comment:
Please change the error massage. Same as the other comments regarding the error message.